composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=7.0",
-        "elasticsearch/elasticsearch": "^5.1.0"
+        "elasticsearch/elasticsearch": "^5.1.0 | ^6.0.0"
```

Comment:
please use `||` as OR operator

(the `|` works for backward compatibility purpose but [it's not documented anymore](https://getcomposer.org/doc/04-schema.md))

( if you have the courage to do it everywhere in the file too, it would be nice, but it's ok if you don't 😄  )

Reply:
Interesting! Thanks for the tip. I'll get right on it.